theano/tensor/nnet/tests/test_conv.py

```diff
@@ -3,7 +3,7 @@
 
 from nose.plugins.skip import SkipTest
 import numpy
-
+import ctypes
```

Comment:
I don't like importing this. We don't do it anywhere else so this add an extra dependency.

Can you use numpy.asarray(3, dtype='uint8') or similar instead?

thanks